src/controllers/update/relevant.period.owned.land.filter.controller.ts

```diff
@@ -26,10 +27,16 @@ export const post = (req: Request, res: Response, next: NextFunction) => {
   try {
     logger.debugRequest(req, `POST ${config.RELEVANT_PERIOD_OWNED_LAND_FILTER_PAGE}`);
     const ownedLandKey = req.body[OwnedLandKey];
+    const appData: ApplicationData = getApplicationData(req.session);
 
     if (ownedLandKey === '1') {
       return res.redirect(config.RELEVANT_PERIOD_INTERRUPT_URL);
     } else {
+      if (appData.update) {
+        appData.update[ChangeBoRelevantPeriodKey] = ChangeBoRelevantPeriodType.NO;
+        appData.update[TrusteeInvolvedRelevantPeriodKey] = TrusteeInvolvedRelevantPeriodType.NO;
+        appData.update[ChangeBeneficiaryRelevantPeriodKey] = ChangeBeneficiaryRelevantPeriodType.NO;
+      }
       return res.redirect(config.UPDATE_FILING_DATE_URL);
```

Comment:
A we spoke about, you can set these to `undefined` and check it's pulling through to mongo.

bear in mind that the `await` call needs to be inside an `async` function so you will need to make this post controller `async`